jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/ContainerActions.kt

```diff
@@ -83,18 +84,28 @@ class ContainerLogsAction(
 
         val window = CloudWatchLogWindow.getInstance(project)
 
-        AwsResourceCache.getInstance(project).getResource(EcsResources.listTaskIds(container.service.clusterArn(), container.service.serviceArn())).thenAccept {
-            val taskId = it.firstOrNull()
-            if (taskId == null) {
-                notifyError(message("ecs.service.logs.no_running_tasks"))
-                return@thenAccept
-            }
-            val logStream = "$logPrefix/${container.containerDefinition.name()}/$taskId"
-            if (!project.awsClient<CloudWatchLogsClient>().checkIfLogStreamExists(logGroup, logStream)) {
-                notifyError(message("ecs.service.logs.no_log_stream"))
-                return@thenAccept
+        AwsResourceCache.getInstance(project)
+            .getResource(EcsResources.listTaskIds(container.service.clusterArn(), container.service.serviceArn()))
+            .thenAccept { tasks ->
+                when {
+                    tasks.isEmpty() -> notifyInfo(message("ecs.service.logs.no_running_tasks"))
+                    tasks.size == 1 && showSingleStream(
+                        window,
+                        logGroup,
+                        "$logPrefix/${container.containerDefinition.name()}/${tasks.first()}"
+                    ) -> return@thenAccept
+
+                }
+                window.showLogGroup(logGroup)
             }
-            window.showLogStream(logGroup, logStream)
+    }
+
+    private fun showSingleStream(window: CloudWatchLogWindow, logGroup: String, logStream: String): Boolean {
+        if (!project.awsClient<CloudWatchLogsClient>().checkIfLogStreamExists(logGroup, logStream)) {
+            notifyInfo(message("ecs.service.logs.no_log_stream"))
```

Comment:
We should update the message to say "showing log streams for the cluster instead" or similar